### PR TITLE
Reviewing Lines Part 6

### PR DESCRIPTION
**Pull request by ttgmichael**
Revisions on QUESTLV100 dialogue lines 2000-2360. Will be covering the next block of dialogue xD

Let me know your thoughts! :)

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -2127,237 +2127,237 @@ QUEST_LV_0100_20150317_002126	{memo X}I will collect the sources of the flame in
...
+QUEST_LV_0100_20150317_002154	$I just feel sorry to my brother.
 QUEST_LV_0100_20150317_002155	$Old Man of Andale Village
-QUEST_LV_0100_20150317_002156	God is looking after our village so our village has clear water and air. And there are no disasters here.
-QUEST_LV_0100_20150317_002157	So finally the Revelator has come.{nl}Very healthy and reliable. I gues we can do it this time.
-QUEST_LV_0100_20150317_002158	Aren't you the Revelator? {nl} So, what brings you to our village? {nl}
-QUEST_LV_0100_20150317_002159	Goddess Saule? {nl} We haven't seen her for a long time. {nl}
-QUEST_LV_0100_20150317_002160	$The reason is that the portal leading to Goddess Saule doesn't open. {nl}I think it's because the Holy Pond is corrupted.
-QUEST_LV_0100_20150317_002161	The Tanus in Zvelgian Vacant Lot carry Purifying Stones. {nl} Those will be able to purify the Pond.
-QUEST_LV_0100_20150317_002162	$Just like there are many demons, the same goes for the goddesses. {nl} The five goddesses are just the ones who became well known.
+QUEST_LV_0100_20150317_002156	$The god looks after our village, so the water is clear. The air is fresh. And there are no disasters.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:2156]: The god -> God

**Good**: But God has helped me to this very day
**Bad**: But the God has helped me to this very day

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -2127,237 +2127,237 @@ QUEST_LV_0100_20150317_002126	{memo X}I will collect the sources of the flame in
...
+QUEST_LV_0100_20150317_002162	$Just like there are many demons, the same goes for the goddesses. {nl}The five goddesses are the only ones who became well known.
 QUEST_LV_0100_20150317_002163	$Our village serves the Goddess Saule, who overlooks the sun. {nl}We used to go see her a lot through the portal, but one day that portal closed.
 QUEST_LV_0100_20150317_002164	{memo X}I haved carved the owls for a long time.{nl}At one point in time, they got their lives and determination.{nl}
 QUEST_LV_0100_20150317_002165	{memo X}When I gained my consciousness, a few hundreds of years have passed.
 QUEST_LV_0100_20150317_002166	{memo X}Carvers should be careful not to exaggerate their skills. They should be humble.
 QUEST_LV_0100_20150317_002167	$It is a good sign that you, the Revelator, came here.{nl}We've been waiting for you for a long time.
 QUEST_LV_0100_20150317_002168	$Please come to our village.{nl}The villagers will really like you.{nl}For sure.
-QUEST_LV_0100_20150317_002169	$The fact that we were able meet was through the goddess' guidance.
+QUEST_LV_0100_20150317_002169	$It was through the god's guidance that we were able to meet.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:2169]: the god -> God's

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -2127,237 +2127,237 @@ QUEST_LV_0100_20150317_002126	{memo X}I will collect the sources of the flame in
...
 QUEST_LV_0100_20150317_002170	$Please refrain from thinking about any suffering. We have all received a blessing.
-QUEST_LV_0100_20150317_002171	Trust the goddess.{nl}We can be together in the tree.
-QUEST_LV_0100_20150317_002172	$We will be saved by sticking together.{nl}The goddess had told us to do that.
-QUEST_LV_0100_20150317_002173	Not yet?{nl}Where is the girl?
+QUEST_LV_0100_20150317_002171	$Trust the god.{nl}We can become one with the tree.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:2171]: Trust the god -> Believe in God.

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -2127,237 +2127,237 @@ QUEST_LV_0100_20150317_002126	{memo X}I will collect the sources of the flame in
...
+QUEST_LV_0100_20150317_002172	$We will be saved by sticking together.{nl}The god told us to do that.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:2172]: The god told us to do that. -> Because God has told us to do that.

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -2127,237 +2127,237 @@ QUEST_LV_0100_20150317_002126	{memo X}I will collect the sources of the flame in
...
-QUEST_LV_0100_20150317_002312	Hm, Okay. I have accessories that will fit on that.{nl}I don't need them anymore so I want to give them to you if you could take care of the monsters nearby.
-QUEST_LV_0100_20150317_002313	Not yet?{nl}They will fit on you nicely.
-QUEST_LV_0100_20150317_002314	Yes. Well done.{nl}Here you go. It's a gift to you as promised.{nl}
-QUEST_LV_0100_20150317_002315	Don't underestimate it. It was a strong animal in the old stories.{nl}In some places, one becomes an enemy to all other people just by touching that animal.
+QUEST_LV_0100_20150317_002308	$Oh, I really am thankful for your help the other time. {nl}My injury is getting better now that I can move. {nl}
+QUEST_LV_0100_20150317_002309	$But, uh, I'm too hungry and I don't have the energy to go back to the captain. {nl}If you have anything to eat, can you share some, please?
+QUEST_LV_0100_20150317_002310	$Oh, thank you so much! {nl}I'll take this to recover and return to the commander for sure.
+QUEST_LV_0100_20150317_002311	$The young birds.. they remind me of Sparnas.{nl}I don't wanna see him even in my dreams.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:2311]: Ellipsis

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**